products/sle/main.pm

```diff
@@ -463,39 +463,19 @@ sub load_slenkins_tests {
 }
 
 sub load_cluster_boot {
-    if (check_var('ARCH', 'aarch64')) {
-        # We need to boot *before* waiting for support server
-        # Because of TianoCore UEFI boot method
-        loadtest "boot/boot_to_desktop";
-        # Wait for support server to complete its initialization
-        loadtest "support_server/wait_support_server";
-        # TODO: poo#32110 add reboot or ensure that network is up
-    }
-    else {
-        # Wait for support server to complete its initialization
-        loadtest "support_server/wait_support_server";
-        loadtest "boot/boot_to_desktop";
-    }
     # Standard boot and configuration
-    loadtest "qa_automation/patch_and_reboot" if is_updates_tests;
-    loadtest "console/consoletest_setup"      if is_updates_tests;
+    boot_hdd_image;
+    loadtest 'qa_automation/patch_and_reboot' if is_updates_tests;
+    loadtest 'console/consoletest_setup' if (is_updates_tests || get_var('HA_CLUSTER'));
 
     return 1;
 }
 
 sub load_ha_cluster_tests {
     return unless (get_var("HA_CLUSTER"));
 
-    # We need to boot *before* waiting for support server
-    # Because of TianoCore UEFI boot method
-    loadtest "boot/boot_to_desktop";
-
-    # Wait for support server to complete its initialization
-    loadtest "support_server/wait_support_server";
-
     # Standard boot and configuration
-    loadtest "qa_automation/patch_and_reboot" if is_updates_tests;
-    loadtest "console/consoletest_setup";
+    load_cluster_boot;
     loadtest "console/hostname" unless is_bridged_networking;
```

Comment:
I know it is not within the scope of PR, but i think that you can remove `unless is_bridged_networking`. We want hostname without condition.